core/indexer/common.go

```diff
@@ -165,7 +166,7 @@ func buildTransaction(
 		SenderShard:   mb.SenderShardID,
 		GasPrice:      tx.GasPrice,
 		GasLimit:      tx.GasLimit,
-		Data:          tx.Data,
+		Data:          string(tx.Data),
```

Comment:
suggestion to encode the data field in hex or base64 as data might contain unprintable characters. Valid on all 3 other casts.

Reply:
done.